README.md

```diff
@@ -1,103 +1,89 @@
-
-**Note:** Gradle projects only have a single `View test report` command. This will open the report if a file can be
-found that matches the configured `tests.reports.html.destination` in the `build.gradle` file, otherwise it will open
-the default `index.html` file.
+![LSP4Jakarta completion](docs/images/LSP4Jakarta-rest-completion.png)
 
-## Contributing
+Code completion when editing microprofile-config.properties files helps you easily set properties for MicroProfile APIs.
 
-Contributions to the Liberty Tools extension are welcome!
+![LSP4MP completion](docs/images/LSP4MP-mp-properties-completion.png)
 
-Our [CONTRIBUTING](CONTRIBUTING.md) document contains details for submitting pull requests.
+Editing assistance for Liberty configuration files is provided through the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server). For more information, see the [project documentation in GitHub](https://github.com/OpenLiberty/liberty-language-server#liberty-config-language-server).
 
-Developing this extension using the
-built-in [gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin/).
+Editing assistance for Jakarta EE APIs in Java files is provided through [Eclipse LSP4Jakarta](https://github.com/eclipse/lsp4jakarta), the Language Server for Jakarta EE. For more information, see the [project documentation in GitHub](https://github.com/eclipse/lsp4jakarta#eclipse-lsp4jakarta).
 
-1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git`
-2. Import this repository as a gradle project in IntelliJ IDEA
-3. Run `./gradlew buildPlugin` to build a `.zip` that can be imported as gradle plugin or run the following Gradle task
-   to build and run an IntelliJ instance:
-   `./gradlew runIde`
+Editing assistance for MicroProfile APIs in microprofile-config.properties and Java files is provided through [Eclipse LSP4MP](https://github.com/eclipse/lsp4mp), the Language Server for MicroProfile. For more information, see the [project documentation in GitHub](https://github.com/eclipse/lsp4mp#eclipse-lsp4mp---language-server-for-microprofile).
 
-### Localization
+## Quick Start
 
-#### LibertyBundles.properties
-Add localized strings in `src/main/resources/messages/LibertyBundles_{locale}.properties`. The default message file is `LibertyBundles.properties`.
+- Install [_Liberty Tools_ from the JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14856-liberty-tools).
+- Projects with the Liberty Maven Plugin or Liberty Gradle Plugin configured will appear in the Liberty tool window on
+  the sidebar. If not enabled by default, the tool window can be viewed by selecting **View > Tool Windows > Liberty**.
+- Select a project in the Liberty tool window to view the available commands.
 
-#### Source code
+For detailed instructions on how to use the Liberty actions check out the [Liberty Tools for IntelliJ IDEA User Guide](docs/user-guide.md) page.
```

Comment:
will the guide include minimum requirements? if so, could call that out here for those that might be looking
```suggestion
For minimum requirements information  and detailed instructions on how to use the Liberty actions, check out the [Liberty Tools for IntelliJ IDEA user guide](docs/user-guide.md) page.
```